src/object/rigid_body.rs

```diff
@@ -614,6 +622,18 @@ impl<N: Scalar> RigidBody<N> {
         self.update_inertia_tensor();
     }
 
+    /// Reference to the collision groups of this rigid body.
+    #[inline]
+    pub fn collision_groups(&self) -> &RigidBodyCollisionGroups {
+        &self.collision_groups
+    }
+
+    /// Mutable reference to the collision groups of this rigid body.
+    #[inline]
+    pub fn collision_groups_mut(&mut self) -> &mut RigidBodyCollisionGroups {
```

Comment:
Yes, immutable access might be useful and has no impact on the physics engine anyway.